apps/editing-toolkit/editing-toolkit-plugin/coming-soon/coming-soon.php

```diff
@@ -77,38 +75,6 @@ function add_public_coming_soon_to_settings_endpoint_post( $input, $unfiltered_i
 }
 add_filter( 'rest_api_update_site_settings', __NAMESPACE__ . '\add_public_coming_soon_to_settings_endpoint_post', 10, 2 );
 
-/**
- * This filter adds the coming soon page to the list of page templates using the `vertical_templates` hook
- *
- * @param array $templates The array of all available page templates.
- *
- * @return array The updated $templates array
- */
-function add_public_coming_soon_page_template( $templates ) : array {
-	$templates[] = (object) COMING_SOON_PAGE_TEMPLATE;
-	return $templates;
-}
-add_filter( 'vertical_templates', __NAMESPACE__ . '\add_public_coming_soon_page_template' );
-
-/**
- * Adds an editable coming soon page to a site
- */
-function add_coming_soon_page_to_new_site() {
-	$coming_soon_template = COMING_SOON_PAGE_TEMPLATE;
-	$new_page_id          = wp_insert_post(
-		array(
-			'post_title'     => $coming_soon_template['post_title'],
-			'post_content'   => $coming_soon_template['post_content'],
-			'post_status'    => 'publish',
-			'post_type'      => 'page',
-			'comment_status' => 'closed',
-			'ping_status'    => 'closed',
-		)
-	);
-	add_option( 'wpcom_public_coming_soon_page_id', $new_page_id );
```

Comment:
There's a reference to the id option below in `coming_soon_page()`

```
	$id = (int) get_option( 'wpcom_public_coming_soon_page_id', 0 );
```

Do we need it?

Reply:
good point, I've just removed that extra code that renders the editable coming soon page